### ISSUE TITLE
Typing lag in recent versions of Terminus

### ISSUE THREAD

**Issue by ryanhugh** [Eugeny/terminus#355]
Looks like the more recent versions of Terminus have significantly more lag when typing in the console. I just installed V35 and it has noticeably less lag. It would be great if the lag could be removed with the latest version of Terminus!

Also want to say great work with Terminus, it is an awesome program & I use it everyday.

**Comment by Eugeny** [comment]
I've just tried v35 and can't notice any difference. Are you using any plugin? (clickable-links specifically)
If you can, please capture a performance trace from DevTools when typing (see https://developers.google.com/web/tools/chrome-devtools/evaluate-performance/reference#save)

**Comment by ryanhugh** [comment]
Thanks for looking into this! I just poked around for an hour or so and it looks like this might be pretty complicated. 

I'm not using any plugins. 

Attached a bunch of files. Some screenshots of dev tools and some performance traces. Looks like, on average, the keydown handlers in v47 are a couple of milliseconds longer than those in v35 but there isn't that big of a difference. I saw a bigger difference in the framerate in v35 vs v47. Here are some screenshots of the FPS counter. Take a look at the distribution of frame rates, to the right of the larger graph in the frame rate layover. It looks like was roughly around 15-20fps in v47 and roughly around 22-25fps in v35. I didn't see any indicator as to why there is this difference, however. 

I'm on Windows 10 and have a NVIDIA GTX 1070 and an i7@3.4GHZ. 

![v47](https://user-images.githubusercontent.com/5777995/40890717-12d3598a-67ad-11e8-8164-02988d73182d.PNG)

![v35](https://user-images.githubusercontent.com/5777995/40890670-6b90f5e2-67ac-11e8-88ec-029dff86a1b9.PNG) 

Let me know if you have any questions!

[files.zip](https://github.com/Eugeny/terminus/files/2066325/typeing.zip)

**Comment by rodg** [comment]
I'm still having this issue on the current build. v35 is noticeably more fluid when typing, and especially so if I hold down a key. Here are two performance traces, one of the v62 and one of v35 (I think I did them correctly). The delay occurs on both my surface pro and my much faster desktop, both running Win10. Both traces were taken on the laptop (I can retake on my desktop if it would help).

Also quick disclaimer, I'm not very familiar with all this yet, so I'm sorry if I'm misplacing something. I actually made this account to comment this issue a few month ago, but my account was randomly flagged so I never got to it.

[traces.zip](https://github.com/Eugeny/terminus/files/2545442/traces.zip)
